Mathlib/CategoryTheory/Yoneda.lean

```diff
@@ -38,6 +38,12 @@ def yoneda : C ⥤ Cᵒᵖ ⥤ Type v₁ where
   map f :=
     { app := fun _ g => g ≫ f }
 
+/-- Variant of the Yoneda embedding which allows a raise in the universe level
+for the category of types. -/
+@[pp_with_univ, simps!]
+def uliftYoneda : C ⥤ Cᵒᵖ ⥤ Type (max w v₁) :=
+  yoneda ⋙ (whiskeringRight _ _ _).obj uliftFunctor.{w}
+
```

Comment:
Do we want/need the following as well?
```lean
def uliftYonedaTrivialIso : uliftYoneda.{0, v₁, u₁} (C := C) ≅ yoneda.{v₁, u₁} (C := C) := sorry
```
to bridge the gap between the two definitions? Hopefully, this could make definitions like `tautologicalCocone` actual special cases (up to a natural isomorphism) of `tautologicalCocone'`.